Makefile

```
@@ -53,7 +54,7 @@ clean:
 ###    Generated Files    ###
 #############################
 get-docker-stacks-upstream-tag:
-	@echo $(DOCKER-STACKS-UPSTREAM-TAG)
+	@echo $(DOCKER-STACKS-UPSTREAM-TAG)$(DOCKER-STACKS-UPSTREAM-DIGEST)
```

Related to tag change above
```suggestion
	@echo $(DOCKER-STACKS-UPSTREAM-TAG)
```